std/functional.d

```diff
@@ -923,6 +916,17 @@ template compose(fun...)
     }
 }
 
+///
+unittest
+{
+    // First split a string in whitespace-separated tokens and then
+    // convert each token into an integer
+    import std.algorithm: equal, map;
+    import std.array: split;
+    import std.conv: to;
```

Comment:
Maybe it's better to move the imports before the comment, so that it's clear that it's referring to the following line, rather than to the imports themselves.